app/components/AddTodoCard.tsx

```diff
@@ -0,0 +1,127 @@
+// components/AddTodoCard.tsx
+import { PlusIcon } from "@heroicons/react/20/solid";
+import React from "react";
+import { useForm } from "react-hook-form";
+import { TodoFormInputsStructure, TodoCardInfoStructure } from "../../types";
+
+export default function AddTodoCard({
+  onAdd,
+}: {
+  onAdd: (item: TodoCardInfoStructure) => void;
+}) {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm({
+    defaultValues: {
+      status: "In progress",
+      type: "Feature",
+      title: "",
+      description: "",
+    },
+  });
+
+  const onFormSubmit = (data: TodoFormInputsStructure) => {
+    const newItem: TodoCardInfoStructure = {
+      id: Date.now(),
```

Comment:
Don't use this kind of solution, instead you can use a library or generate a random ID yourself